deepmd/dpmodel/train/validation.py

```
@@ -0,0 +1,568 @@
+        if not self.should_run(display_step):
+            return None
+
+        result: FullValidationResult | None = None
+        caught_exception: Exception | None = None
+        error_message = None
+        save_path: str | None = None
+        if self.rank == 0:
+            try:
+                result = self._evaluate(display_step)
+                save_path = result.saved_best_path
+            except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)